src/pages/apply/project-manager/ApplyProjectManager.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { makeStyles } from '@material-ui/core/styles';
+import { Container, Box, Typography, Grid } from '@material-ui/core';
+import Hero from '../../../components/Hero';
+import { ReactComponent as SVG } from './projectmanager.svg';
+import { TextQuoteBlock } from '../../../components/TextQuoteBlock';
+import CTA from '../CTA';
+// @ts-ignore
+import Fade from 'react-reveal/Fade';
+
+const useStyles = makeStyles({
+  image: {
+    width: '100%',
+    borderRadius: 10,
+    margin: 'auto',
+  },
+  list: {
+    marginTop: 0,
+    '& > li': {
+      marginBottom: '0.25em',
+    },
+  },
+  stepper: {
+    paddingTop: '2.5em',
+    paddingRight: '0',
+    maxWidth: '16em',
+  },
+  application: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-end',
+  },
+  text: {
+    //fontFamily: '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif',
```

Comment:
Why is this commented out? Just remove if not necessary?

Reply:
Not sure, this is copied from the software developer page, @rymaju ?